test_ros2cli/test/config_ros2action_test.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
@ivanpauno both here and in the next `config_*.py` file, I wonder if we can aggregate the information that's currently distributed in multiple collections into a single one so that's easier to read and extend. For instance, a list of dictionaries that keep verb, flags, output, etc. What do you think?